plugins/org.eclipse.n4js.ui/src/org/eclipse/n4js/ui/organize/imports/ImportsComputer.xtend

```diff
@@ -4,90 +4,93 @@
  * are made available under the terms of the Eclipse Public License v1.0
  * which accompanies this distribution, and is available at
  * http://www.eclipse.org/legal/epl-v10.html
- *
+ * 
```

Comment:
trailing white space

Reply:
Xtend autoformatter is doing that :/
I will remove it, but this will come back...

Reply:
Turn off Xtend auto-formatter 👯